packages/web/src/pages/saved-page/components/desktop/SavedPage.tsx

```diff
@@ -122,6 +124,11 @@ const SavedPage = ({
 }: SavedPageProps) => {
   const { mainContentRef } = useContext(MainContentContext)
   const initFetch = useSelector(getInitialFetchStatus)
+
+  const { isEnabled: isPremiumEnabled } = useFeatureFlag(
+    FeatureFlags.PREMIUM_ALBUMS_ENABLED
+  )
```

Comment:
i think we can just remove this ff usage since we already launched the premium albums feature.